packages/proxy/README.md

```diff
@@ -0,0 +1,3 @@
+# proxy
+
+This package contains the code for Cypress's HTTP interception proxy.
```

Comment:
Same README instructions needed as noted in `network` package.

Reply:
Added in c90e72b11